numpy/linalg/_linalg.py

```diff
@@ -1721,10 +1705,6 @@ def svd(a, full_matrices=True, compute_uv=True, hermitian=False):
     Notes
     -----
 
-    .. versionchanged:: 1.8.0
-       Broadcasting rules apply, see the `numpy.linalg` documentation for
-       details.
-
```

Comment:
Rather than delete these mis-formatted lines, they should be aligned with the surrounding text